### ISSUE TITLE
Running on Fargate task & locally seams docker-entrypoint.sh is not executed

### ISSUE THREAD

**Issue by Gigiart** [codedge/aws-fargate-with-php#4]
First of all this is a great job. Thank you to sharing with us - special beginners.
 
However, running the Dockerfile on Fargate task & locally 

`docker run -it -p 80:80 <container>`

It seams docker-entrypoint.sh is not executed and, following that, I've got 502 bad gateway from nginx as php-fpm has not started.

Thanks

**Comment by codedge** [comment]
I think it would better to run `docker-compose up -d`. That uses [docker compose](https://docs.docker.com/compose/) for orchestrating the container.

**Comment by ghost** [comment]
Yes, but network is not setup in the `docker-compose.yml` file, so containers do not see each other
